providers/push-webhook/package.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "@novu/push-webhook",
+  "version": "0.14.0",
+  "description": "A push-webhook wrapper for novu",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "private": false,
+  "repository": "https://github.com/novuhq/novu",
+  "license": "MIT",
+  "keywords": [],
+  "scripts": {
+    "prebuild": "rimraf build",
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
```

```suggestion
    "fix:lint": "eslint src --ext .ts --fix",
    "lint": "eslint src --ext .ts",
```
We need to add a `lint` command. 🙏🏻